lib/stats-collector.js

```diff
@@ -1,8 +1,15 @@
 'use strict';
 
+/**
+ * Provides a factory function for a {@link StatsCollector} object.
+ * @private
+ * @module
+ */
+
 /**
  * Test statistics collector.
  *
+ * @private
```

Comment:
This would be `@public` since this is how reporters (both ours and 3rd party) get the statistics from the run. But the tag is not meaningful in a @typedef.